rendezvous/README.md

```diff
@@ -2,7 +2,7 @@
 
 | Lifecycle Stage | Maturity      | Status | Latest Revision |
 |-----------------|---------------|--------|-----------------|
-| 1A              | Working Draft | Active | r3, 2021-07-12  |
+| 1A              | Working Draft | Active | r3, 2022-12-20  |
```

Comment:
None of the below changes semantics, right? If so, I don't think the revision bump is needed here.

Reply:
Sorry, this was accidental (lint tool).